info-turing/index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Turing Info Session
+layout: lesson
+---
+
+# Turing Info Session
+
+## Welcome!
+
+We are so glad you are here! To get set up...
+1. Please change your Zoom name to first name, last initial, and pronouns, like this: _Kayla G. (she/her)_. Not sure how to do that? Hover over your image in the Zoom gallery, click the three dots, and select Rename.
+2. Heads up! We will ask you to briefly introduce yourself in a minute.
+
+## Goals
+
+The goal of this session is to give you a general overview of Turing:
+- what we teach and why
+- what it's like to be a student here
+- what support we offer for the job hunt
+
+## Online Learning Norms
+
+- Ask lots of questions! This session is for YOU, so don't hesitate to speak up!
+- Mute your microphone unless you’re the main speaker.
+- Keep your camera on during class (if possible).
+- Disconnected? Jump back on!
+
+## Zoom Tools
+
+Want to get our attention during the workshop? Raise your hand or type your question in the chat! Let's practice:
+- **Raise Hand**: Find the reactions button in your Zoom toolbar and raise your hand.
+- **Zoom Chat**: Enter your answer in the Zoom chat - How much do you know about Turing, the school? `Absolutely nothing`, `I have a vague idea`, `I know all about it!`, or something else!
+- **Unmute**: Introduce yourself! Please share your name, pronouns, location, and what you're hoping to get out of this session.
+
+## Workshop Overview:
+- [Who Is Turing?](#who-is-turing?)
+- [What We Teach](#what-we-teach)
+- [How We Teach](#how-we-teach)
+- [Professional Development and Job Hunt Support](#professional-development-and-job-hunt-support)
+- [Your Questions Here](#your-questions-here)
+- [Next Steps](#next-steps)
```

Comment:
Love the layout! Having bookmarks to easily navigate the page, while having everyone on one page is GREAT